### DISCUSSION TITLE
Pydantic compatibility

### DISCUSSION THREAD

**Discussion by 07pepa** [encode/httpx#3142] [Ideas]
It would be nice to have httpx compatible with Pydantic (we can start with url that is a lot of times stored in settings) se what needs to be added here https://github.com/pydantic/pydantic-extra-types/pull/156. please respond quickly

**Comment by MarcBresson** [comment]
Hello, I can tackle that problem :)

**Comment by MarcBresson** [comment]
GitHub does not seem to automatically add links so there you go : https://github.com/encode/httpx/pull/3450
